pkgs/development/tools/ocaml/opam/default.nix

```diff
@@ -122,6 +122,11 @@ in stdenv.mkDerivation {
   outputs = [ "out" "installer" ];
   setOutputFlags = false;
 
+  # Work around https://github.com/NixOS/nixpkgs/issues/166205.
+  env = lib.optionalAttrs stdenv.cc.isClang {
+    NIX_LDFLAGS = "-l${stdenv.cc.libcxx.cxxabi.libName}";
```

Comment:
this links `libc++abi.dyl-whatever` into all linked objects. unfortunately, that adds c++abi to opam-installer, which seems undesirable given it doesn't link to libc++.

Reply:
Good point.
I'll merge this PR since it seems to be affecting multiple users, and I'll take a proper look after holidays.